### DISCUSSION TITLE
error.

### DISCUSSION THREAD

**Discussion by djnbeeman** [Leather128/LeathersFunkinEngine#149] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/88124585/149702506-209afee7-7ee5-4cc1-9d69-cad499bbee9c.png)
I got this error when I didn't even edit anything in the assets folder(well I did because of the default difficulties and decided to add my own difficulties and then I edited the week out of it). @Leather128

**Comment by Leather128** [comment, marked as answer]
well clearly you modified the week list to include a week that didn't exist, keep in mind any custom weeks or related files must be in a mod folder unless you're putting them in the assets folder from the ***source code***

**Reply by djnbeeman** [reply]
![image](https://user-images.githubusercontent.com/88124585/149817546-d0a612d4-c02e-4090-a28d-580a86036992.png)
![image](https://user-images.githubusercontent.com/88124585/149817582-f36498f1-1142-4e6b-8599-46378c282e57.png)

**Reply by djnbeeman** [reply]
nvm fixed just had the week as the mod name.
